tests/migration/migration_setup.pm

```diff
@@ -0,0 +1,25 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: zypper
+# Summary: A perparation on repo before migration
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use migration;
+use qam;
+
+sub run {
+    select_console 'root-console';
+
+    disable_installation_repos;
+    set_zypp_single_rpmtrans;
```

Comment:
from which product is this available?
`ZYPP_SINGLE_RPMTRANS`.

We don't run it either, so we have two empty actions here, which mean we don't need the test module to be scheduled.
And for the cases where this is needed, a description in the test suite should make it clear, actually we should not even use that setting above but just schedule the module.

Reply:
I removed ZYPP_SINGLE_RPMTRANS, it is not necessary in migration cases.

Reply:
I guess it could be used somehow...right? but why to improve the performance, anyway we will have autoyast almost 0 time of patching, we are not testing patching :) agreed.